src/core/fonts.js

```diff
@@ -1073,6 +1074,14 @@ class Font {
       nonStdFontMap[fontName] && stdFontMap[nonStdFontMap[fontName]]
     );
 
+    if (isStandardFont) {
+      const stdFontMetrics = getStdFontMetrics();
+      const metrics = stdFontMetrics[fontName];
+      this.ascent = metrics.ascent / PDF_GLYPH_SPACE_UNITS;
+      this.descent = metrics.descent / PDF_GLYPH_SPACE_UNITS;
+      this.capHeight = metrics.capHeight / PDF_GLYPH_SPACE_UNITS;
```

Comment:
Is it *generally* appropriate to overwrite possibly existing values, based on the font dictionary, with `NaN` as will happen for some of the fonts (e.g. ZapfDingbats)?

Reply:
I think there are no such metrics in pdb files but maybe I'm wrong.
Anyway I got the different values from:
https://github.com/Hopding/standard-fonts/tree/master/font_metrics

Reply:
> I think there are no such metrics in pdb files but maybe I'm wrong.

Please note that those values (initially) come from the /FontDescriptor dictionary in the PDF document itself (although for embedded font files we then override those after parsing the font data), via https://github.com/mozilla/pdf.js/blob/6b9cc24d49583c487d9267c98579fd4cd9db1704/src/core/fonts.js#L933-L936 and https://github.com/mozilla/pdf.js/blob/6b9cc24d49583c487d9267c98579fd4cd9db1704/src/core/evaluator.js#L4063-L4066